rotkehlchen/tests/unit/test_globaldb.py

```diff
@@ -265,3 +266,44 @@ def test_enum_values_are_present_in_global_db(globaldb, enum_class, table_name):
     for enum_class_entry in enum_class:
         r = cursor.execute(query, (enum_class_entry.value,))
         assert r.fetchone() == (1,), f'Did not find {table_name} entry for value {enum_class_entry.value}'  # noqa: E501
+
+
+def test_global_db_restore(globaldb):
+    """
+    Check that the user can recreate assets information from the packaged
+    database with rotki. The test adds a new asset, restores the database
```

Comment:
Augment this test with at least 1 non ethereum asset, and also with an ethereum token with underlying tokens so that the test can be catching all tables.